requirements.txt

```diff
@@ -17,7 +17,7 @@ urllib3>=1.23,<1.25
 PyYAML>=5.1
 jinja2>=2.10
 configparser>=3.5.0
-openshift>=0.8.6
+openshift>=0.8.6,<0.8.8
```

Comment:
@14rcole Any reason for the changing openshift version ? 
what was the error/bug?

Reply:
There's another package that requires urllib <1.25, and openshift 0.8.8 requires urllib>=1.25.  I forget exactly which package.  I'll figure it out and make a note of it so we know when we can update openshift

Reply:
Update: it looks like we require urllib<1.25.  So I can look into updating that as well at some point